common/testData/localeIdentifiers/localeDisplayName.txt

```diff
@@ -1310,6 +1310,32 @@ nl-Latn-BE; flamšćina (łaćonsce)
 zh-Hans-fonipa; chinšćina [zjednorjena] (FONIPA)
 
 
+@locale=ht
```

Comment:
The Haitian Creole data here is added because it wasn't added when the locale xml was added. It was created by the automated script that I run to rebuild files that inherit from the population data.

`mvn package -DskipTests=true &&  java -jar tools/cldr-code/target/cldr-code.jar ConvertLanguageData &&  java -jar tools/cldr-code/target/cldr-code.jar GenerateLikelySubtags &&  java -jar tools/cldr-code/target/cldr-code.jar GenerateTestData`

Reply:
It's fine, of course test data is generated as part of the BRS cycle, it does not strictly need to be added when the locale was added.